doc-surrealdb_versioned_docs/version-latest/surrealql/functions/database/string.mdx

```diff
@@ -806,6 +810,24 @@ true
 
 <br />
 
+## `string::is::record`
+
+The `string::is::record` function checks whether a string is a Record ID.
+
+```surql title="API DEFINITION"
+string::is::record(string, option<string | table>) -> bool
+```
```

Comment:
```suggestion
```
::: note
The second argument is optional and can be used to specify the table name that the record ID should belong to. If the table name is provided, the function will check if the record ID belongs to that table only.
:::
```